pkgs/tools/misc/mslink/default.nix

```diff
@@ -17,6 +17,9 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
+    if [[ "$(uname)" == "Darwin" ]]; then
+      mv mslink.exe mslink
+    fi
     install -D mslink $out/bin/mslink
   '';
 
```

Comment:
Can you update the homepage to ``https://www.mamachine.org/mslink/index.en.html``.